tasks-by-users/AbbasShaikh0304/05-Css Grid Layout/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>CSS Grid Layout</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="header">Header</div>
+      <div class="content-large">Content #1</div>
+      <div class="content-small">Content #2</div>
+      <div class="content-small">Content #3</div>
+      <div class="content-long">Content #4</div>
+      <div class="footer">Footer</div>
```

Comment:
use footer tag

Reply:
Thank you for your suggestion. Commited the changes. @AlishaPrasad